src/Select.js

```diff
@@ -659,7 +691,7 @@ var Select = React.createClass({
 
 		var ops = Object.keys(this.state.filteredOptions).map(function(key) {
 			var op = this.state.filteredOptions[key];
-			var isSelected = this.state.value == op.value;
+			var isSelected = this.state.value === op.value;
```

Comment:
For the record I _didn't_ need to change this except running `npm run lint` picks it up, and that was already in the `master` branch (https://github.com/JedWatson/react-select/blob/master/src/Select.js#L662) so someone didn't run it before committing :stuck_out_tongue: